test/ServerComparison.FunctionalTests/Helpers.cs

```diff
@@ -10,8 +10,13 @@ public class Helpers
     {
         public static string GetApplicationPath(ApplicationType applicationType)
         {
-            return Path.GetFullPath(Path.Combine("..", "..", "..", "..", "..", 
-                    applicationType == ApplicationType.Standalone? "ServerComparison.TestSites.Standalone" : "ServerComparison.TestSites"));
+#if NET451
```

Comment:
To prevent this craziness, probably you could do something similar to what MVC does...it walks up the path to find the directory having the solution file and then builds up the path as needed...
https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.TestCommon/SolutionPathUtility.cs